packages/owlbot-bootstrapper/common-container/test/split-repo.test.ts

```diff
@@ -114,32 +114,52 @@ describe('SplitRepo class', async () => {
   });
 
   it('should create an empty git repo on disk', async () => {
+    const splitRepo = new SplitRepo(
+      'python' as Language,
+      'google.cloud.kms.v1',
+      octokit,
+      'ghs_1234'
+    );
+
     await splitRepo._initializeEmptyGitRepo('python-kms-1', directoryPath);
     assert.ok(fs.statSync(`${directoryPath}/${splitRepo.repoName}-1`));
     assert.ok(fs.statSync(`${directoryPath}/${splitRepo.repoName}-1/.git`));
   });
 
   it('should push changes in an empty repo to github', async () => {
-    await splitRepo._initializeEmptyGitRepo(FAKE_REPO_NAME, directoryPath);
-    fs.writeFileSync(`${directoryPath}/${FAKE_REPO_NAME}/README.md`, 'hello!');
+    const splitRepo = new SplitRepo(
+      'python' as Language,
+      `google.cloud.${FAKE_REPO_NAME}.v1`,
+      octokit,
+      'ghs_1234'
+    );
+
+    await splitRepo._initializeEmptyGitRepo(
```

Comment:
nit: generally we don't want to call private methods in tests -- it makes the implementation and tests fragile.